CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # bedrock-account-http ChangeLog
 
+## 8.0.0 - 2023-10-xx
+
+### Changed
+- **BREAKING**: Drop support for Node.js < 18.
+- Add `@bedrock/turnstile` dep. This module requires Node.js 18+.
+
+### Added
```

Comment:
`Added` should come before `Changed` (we alphabetize for quick skimming / finding of stuff in the changelog).